app/components/ui/tale-tab-metadata/template.hbs

```diff
@@ -38,6 +38,24 @@
         </div>
     </div>
 
+    <div class="inline fields ui grid">
+        <label class="two wide column right aligned">License</label>
+        <div class="field thirteen wide column">
+            <div id="licenseDropdown" class="ui icon selection dropdown license {{if (eq licenses.length 0) "loading"}}">
```

Comment:
Adding the following to `main.scss` should make the width of this field the same as the others on the form:
```css
#licenseDropdown {
    width: 100%;
}
```

Reply:
@ThomasThelen  please make this change before merging